components/support/rc_crypto/src/lib.rs

```diff
@@ -52,8 +52,8 @@ pub use ece;
 
 pub use crate::error::{Error, ErrorKind, Result};
 
-/// Only required to be called if you intend to use this library in conjunction
-/// with the `hawk` or the `ece` crate.
+/// Only required to be called if you intend to either use this library in conjunction with the
+/// `hawk` or the `ece` cratem, or use any functionality based on nss.
```

Comment:
nit: typo "cratem" - but it seems these docs could be tightened up - IIUC, the only time people use this crate is specifically for nss, so in practice I think it's now "you must call this"?